src/listener/ov-listener.js

```diff
@@ -62,7 +62,15 @@ export default function(robot, client) {
 
     let checkedMessage = filter.check(resource);
     if (typeof checkedMessage !== 'undefined' && checkedMessage.length > 0) {
-      transform.messageRoom(client.notificationsRoom, resource.resource);
+      let room = '';
```

Comment:
Can you shorten this to:

let room = '#' + client.notificationsRoom;
if (robot.adapterName === 'flowdock') {
  room = client.notificationsRoom;
}

Reply:
Great, fixed.